uber/automated_emails.py

```diff
@@ -1,42 +1,60 @@
-# WARNING - changing the email subject line for an email causes ALL of those emails to be re-sent!
-# Note that since c.EVENT_NAME is used in most of these emails, changing the event name mid-year
-# could cause literally thousands of emails to be re-sent!
-
 from uber.common import *
 
 
 class AutomatedEmail:
+    """
+    Represents one category of emails that we send out.
+
+    An example of an email category would be "Your registration has been confirmed".
+    Each category
```

Comment:
documentation, amirite? :)   removed the extra comment because now this class is documented like crazy elsewhere